user-service/cmd/user-server/main.go

```diff
@@ -1,45 +1,20 @@
-package main
-
-import (
-	"context"
-	"fmt"
-
-	"github.com/go-redis/redis/v8"
-)
-
-var ctx = context.Background()
-
-func ExampleClient() {
-	rdb := redis.NewClient(&redis.Options{
-		Addr:     "localhost:6379",
-		Password: "", // no password set
-		DB:       0,  // use default DB
-	})
-
-	err := rdb.Set(ctx, "key", "value", 0).Err()
-	if err != nil {
-		panic(err)
-	}
-
-	val, err := rdb.Get(ctx, "key").Result()
-	if err != nil {
-		panic(err)
-	}
-	fmt.Println("key", val)
-
-	val2, err := rdb.Get(ctx, "key2").Result()
-	if err == redis.Nil {
-		fmt.Println("key2 does not exist")
-	} else if err != nil {
-		panic(err)
-	} else {
-		fmt.Println("key2", val2)
-	}
-	// Output: key value
-	// key2 does not exist
-}
-
-func main() {
-	fmt.Println("User server")
-	ExampleClient()
-}
+package main

```

Comment:
Putting this comment here even though it's not related to this file.

You shouldn't include `dump.rdb` files (both the one at `cmd/user-server` and at the root) to version control systems such as `git`

Take a look at how to add a `.gitignore` file and how to specify ignoring `.rdb` files